test/components/header.html

```diff
@@ -48,40 +34,45 @@
                   <img class="ui icon" data-icon="text_big" />
                 </div>
               </div>
-      
+    
               <div class="darkmode">
                 <div class="light-button" onclick="contrast_light()">A</div>
                 <div class="dark-button" onclick="contrast_dark()">A</div>
               </div>
-
-              <div class = "page-options">
+    
+              <div class="page-options">
                 <span class="language">
                   <a>&#x0939;&#x093F;&#x0928;&#x094D;&#x0926;&#x0940;</a>
                   &#9474;<a>Eng</a></span>
-              </div>                
-
-            </div>
-            <hr class="divider">
-            <div class="page-options">
-              <a href="https://iitr.ac.in/Institute/Contact%20Us.html">Contact Us</a>
-              <a href="https://iitr.ac.in/support/td/">Directory</a>
-              <a href="https://iconnect.iitr.ac.in/">iConnect</a>
-              <a href="https://newwebmail.iitr.ac.in/">Webmail</a>
-            </div>
-          </div>
-          <div class="right-container">
-            <div class="page-search">
-              <img class="ui icon" data-icon="search" onclick="open_search()" />
+              </div>
+              <div class="page-search">
+                <img class="ui icon" data-icon="search" onclick="open_search()" />
+              </div>
             </div>
           </div>
+           
+         
+        </div>
+      
+      <div class="ui main-head">
+        <div id="hamBurger" class="hamburger-container">
+          <img class="ui icon" data-icon="hamburger" />
+        </div>
+        <div class="logo">
+          <img onclick='window.location="https://iitr.ac.in"'
+            src="https://cmsredesign.channeli.in/library/assets/images/iitrLogo.png" alt="" width="100%" />
+        </div>
+        <div class="g20-logo">
+          <img onclick='window.open("https://www.g20.org/en/", "_blank")'
+            src="https://cmsredesign.channeli.in/library/assets/images/g20-logo.png" alt="" width="100%" />
         </div>
       </div>
-    </div>
-    
-    </body>
+      </div>
+  </body>
+  
 
   <script src="http://cmsredesign.channeli.in/library/javascript/components/menu.js?rndstr='22SmQfdqa8'"></script>
-  <script src="http://cmsredesign.channeli.in/library/javascript/components/header.js?rndstr='22SmQfdqa8'"></script>
+  <script src="http://localhost:5500/codebase/chakra-library/library/javascript/components/header.js?rndstr='22SmQfdqa8'"></script>
```

Comment:
remove this or fix this file, no one will be able to open this while testing

Reply:
Done